spec/support/shared_examples/abilities.rb

```diff
@@ -0,0 +1,95 @@
+# collections of shared examples
+
+# logged in, unconfirmed:
+shared_examples "same as guest user" do
+  it_behaves_like "can view public pages"
+  it_behaves_like "can not modify things on public pages"
```

Comment:
Small stylistic suggestion: maybe use `cannot` instead of `can not`? I think it makes it easier to parse out negatives and matches CanCanCan's `cannot`.

Reply:
Hmmm They are not always used the same as cancancan's `cannot`'s. That made it more confusing, imo, so I left them as is.